lib/karen.coffee

```diff
@@ -45,6 +45,7 @@ class MockScreen
 class MockElement extends Evented
   constructor: (@type) ->
     super()
+    @__defineGetter__ 'tagName', -> @type
```

Comment:
It is not clear to me what implementing HTMLUnknownElement would accomplish? According to MDN: "Properties; No specific property; inherits properties from its parent, HTMLElement. Methods; No specific method; inherits properties from its parent, HTMLElement," so unless someone is explicitly checking, there is no difference. Note that most Mock* are directly constructed by tests in agency and do not pass through .createElement() and so a separate `MockUnknownElement` or similar will not help.